cli/flow-typed/npm/jest_v23.x.x.js

```diff
@@ -958,7 +1102,13 @@ declare var expect: {
   objectContaining(value: Object): Object,
   /** Matches any received string that contains the exact expected string. */
   stringContaining(value: string): string,
-  stringMatching(value: string | RegExp): string
+  stringMatching(value: string | RegExp): string,
+  not: {
+    arrayContaining: (value: $ReadOnlyArray<mixed>) => Array<mixed>,
+    objectContaining: (value: {}) => Object,
+    stringContaining: (value: string) => string,
+    stringMatching: (value: string | RegExp) => string,
+  },
```

Comment:
I was working on tests so thought that up to date jest libdefs would make it easier :) Definitely would've been easier to have separate PR for updating libdefs. And to highlight just in case, these are CLI libdefs, not the ones users downloads